Tasks/PublishSymbolsV2/PublishSymbols.ps1

```diff
@@ -177,6 +177,7 @@ try {
         [string]$asAccountName = (Get-VstsTaskVariable -Name 'ArtifactServices.Symbol.AccountName')
         [string]$PersonalAccessToken = (Get-VstsTaskVariable -Name 'ArtifactServices.Symbol.PAT')
         [bool]$UseAad = (Get-VstsTaskVariable -Name 'ArtifactServices.Symbol.UseAad' -AsBool)
+        [string]$IndexableFileFormats = (Get-VstsInput -Name 'IndexableFileFormats' -Default 'Default')
```

Comment:
I think having this be empty string rather than 'Default' would make it backward compatible.  Meaning if indexableFileFormats is not present it will not add the new parameter to the commandline.  After this change it will always add the new command line parameter with the value of 'Default' for TeamFoundation type symbol publish.

Reply:
I guess I'm worried that somehow we wind up executing a downlevel symbol.exe with this code and it would break because it doesn't recognize the new command line parameter.